codespell_lib/data/dictionary.txt

```diff
@@ -9234,6 +9258,7 @@ entitlied->entitled
 entitys->entities
 entoties->entities
 entoty->entity
+entquiry->enquiry
```

Comment:
Yikes, this is a can of worms:  
https://www.grammarly.com/blog/inquire-enquire/

```
entquire->enquire, inquire,
entquires->enquires, inquires,
etquiries->enquiries, inquiries,
```

Reply:
Removed